poollet/machinepoollet/controllers/machine_controller_networkinterface.go

```diff
@@ -376,16 +376,17 @@ func (r *MachineReconciler) convertIRINetworkInterfaceState(state iri.NetworkInt
 	return "", fmt.Errorf("unknown network interface attachment state %v", state)
 }
 
-func (r *MachineReconciler) convertIRINetworkInterfaceStatus(status *iri.NetworkInterfaceStatus) (computev1alpha1.NetworkInterfaceStatus, error) {
+func (r *MachineReconciler) convertIRINetworkInterfaceStatus(status *iri.NetworkInterfaceStatus, machine *computev1alpha1.Machine, machineNic computev1alpha1.NetworkInterface) (computev1alpha1.NetworkInterfaceStatus, error) {
```

Comment:
What do you think about passing the `NIC` name instead of `machine` & `machineNic` since both props are only used to calculate the name itself.

Reply:
done